app/controllers/smart_proxies_controller.rb

```diff
@@ -1,5 +1,8 @@
 class SmartProxiesController < ApplicationController
   include Foreman::Controller::AutoCompleteSearch
+  include Foreman::Controller::ExtensiblePage
```

Comment:
I'd rather use `PageletSupport` so we get rid of Page term completely which is quite misleading.